manager/controlapi/service.go

```diff
@@ -141,6 +141,18 @@ func validateContainerSpec(container *api.ContainerSpec) error {
 	return nil
 }
 
+func validatePluginSpec(plugin *api.PluginSpec) error {
+	if plugin.Image == "" {
+		return grpc.Errorf(codes.InvalidArgument, "PluginSpec: image reference must be provided")
+	}
+
+	if _, err := reference.ParseNormalizedNamed(plugin.Image); err != nil {
```

Comment:
@aaronlehmann 
> IIRC yes. The tag is mandatory, but you can also provide a digest

Is this line correct then? I assume it will allow tag-less images.

@anusha-ragunathan @tiborvass Can you confirm tags are mandatory for plugins?